k8s/deployment.yml

```diff
@@ -109,6 +109,8 @@ spec:
         volumeMounts:
         - name: jibri-metadata-updater
           mountPath: /opt/jibri-metadata-updater
+      nodeSelector:
```

Comment:
should you change the file name as well?

Reply:
I don't understand what you are suggesting. Can you please give me more details?

Reply:
I understood that you are preparing for the arrival of jitsi's other services. The file name is `deployment.yml` but you will have other services now so the name may need to be more precise?

Reply:
Ok, thanks, I see! The purpose of this PR is to separate jibri from other services in distinct nodepools, in order to solve a cluster autoscaling issue.

But I totally agree with you: these kubernetes resource definition files should be renamed, their name are too generic.
I'll do it in another PR because all files in the `k8s` directory are concerned and i'm thinking about another project structure, more modular and easier to override with custom values.